apps/arweave/src/ar_header_sync.erl

```diff
@@ -280,6 +280,8 @@ handle_info({event, disksup, {remaining_disk_space, "default", true, _Percentage
 		true ->
 			case State#state.is_disk_space_sufficient of
 				true ->
+					ok
+				false ->
```

Comment:
I believe in this case `is_disk_space_sufficient` is being used as a toggle to limit the number of time that the warning is printed. It should only be printed once, the first time that `Bytes < DiskPoolSize + DiskCacheSize + BufferSize div 2`. Once it prints it once it sets `is_disk_space_sufficient` false and won't print again until `is_disk_space_sufficient` is set to `true` and then `false`

Current logic is correct I believe.